plugin-qeyes/qeyesimagewidget.cpp

```diff
@@ -32,16 +32,21 @@
 #include "qeyesimagewidget.h"
 
 
-bool ImageStretcher::load(QString fn) {
-    if (fn.toLower().endsWith(QString::fromUtf8(".svg"))) {
+bool ImageStretcher::load(const QString& fn)
+{
+    if (fn.endsWith(QString::fromUtf8(".svg"), Qt::CaseInsensitive))
+    {
         svg = true;
         if (!svgrender.load(fn))
             return false;
-    } else {
+    }
+    else
+    {
```

Comment:
Are these style changes intentional?

Reply:
Personally I think putting braces on newline makes it more readable.
We don't have clang format so style is not consistent.